src/main/java/io/openliberty/tools/intellij/util/LibertyMavenUtil.java

```diff
@@ -246,8 +249,8 @@ private static String getLocalMavenWrapper(VirtualFile buildFile) throws Liberty
      * @return Maven path to be executed or an exception to display
      * @throws LibertyException
      */
-    private static String getCustomMavenPath(Project project, String customMavenHome) throws LibertyException {
-        File mavenHomeFile = MavenUtil.resolveMavenHomeDirectory(customMavenHome); // when customMavenHome path is invalid it returns null
+    private static String getCustomMavenPath(Project project, StaticResolvedMavenHomeType customMavenHome) throws LibertyException {
+        File mavenHomeFile = MavenUtil.getMavenHomeFile(customMavenHome); // when customMavenHome path is invalid it returns null
```

Comment:
Instead of changing parameter type to StaticResolvedMavenHomeType, can we revert it back to string and is it possible to call "MavenUtil.getMavenHomeFile" method in a similar way written inside the "MavenUtil.resolveMavenHomeDirectory" deprecated method.

@Nullable
@Deprecated(forRemoval = true)
public static File resolveMavenHomeDirectory(@Nullable String overrideMavenHome) {
  if (!isEmptyOrSpaces(overrideMavenHome)) {
    //noinspection HardCodedStringLiteral
    return MavenUtil.getMavenHomeFile(staticOrBundled(resolveMavenHomeType(overrideMavenHome)));
  }